### PR TITLE
New PR, with fixed conflicts and minor changes

### PR DESCRIPTION
**Pull request by Dhruv-m-Shah**
- When a user submits the application, results are stored in a DynamoDB table
- Github_url, Linkedin_url, Dribble_url, personal Url, and resume url are no longer required
- Fixed bug when url sections were left blank.

### CODE REVIEW

File: src/views/ApplicationPage/index.js [hunk 1]
```diff
@@ -16,6 +16,17 @@ import SubmitButton from "../../components/SubmitButton";
...
+let awsConfig = {
+  region: "us-east-1",
+  endpoint: "http://dynamodb.us-east-2.amazonaws.com",
+  accessKeyId: process.env.REACT_APP_AWS_ACCESS_KEY,
+  secretAccessKey: process.env.REACT_APP_AWS_SECRET_KEY,
+};
+
+AWS.config.update(awsConfig);
+
 export default class application extends Component {
```
Comment (dunja-tomic) [inline src/views/ApplicationPage/index.js:30]: capitalize "Application" for consistency

File: src/views/ApplicationPage/index.js [hunk 2]
```diff
@@ -72,40 +83,84 @@ export default class application extends Component {
...
       document.getElementById("display_error").innerHTML =
         "Over 1000 character limit for 'Why Golden Hack'";
     }
 
     this.setState({ submitted: true });
 
     console.log(this.state);
+    let docClient = new AWS.DynamoDB.DocumentClient();
+    let input = {
+      email: "dh", // How do i get the email from JWT
```
Comment (dunja-tomic) [inline src/views/ApplicationPage/index.js:136]: Take a look at how it was done here -> https://github.com/thegoldenhack/website-2020/blob/94f170cab16a7f777f12763bc69693832be89efa/src/views/DashboardPage/index.js#L56

File: src/views/ApplicationPage/index.js (hunk 1 above)
Comment (Dhruv-m-Shah) [reply src/views/ApplicationPage/index.js]: Done!

**Comment by Dhruv-m-Shah** [comment]
Fixing merge conflicts

**Pull request closed without merging**